custom_components/tuya_local/devices/dr_heater_dr008_thermostat.yaml

```diff
@@ -0,0 +1,229 @@
+name: Dr. Heater DR-008 Electric Radiant Floor Thermostat
```

Comment:
This name should be under the products section, along with the product id.

The top level name should be generic, without any branding, as the same device is often sold in different markets with different branding, and this is the one that is used for device naming (which is also a reason to keep it short).

I think "Floor thermostat" would be appropriate for the top level name, and "Dr. Heater DR-008" for the product specific name.

Reply:
Okay, I'll update